packages/react-components/src/components/BrowserPermissionDeniedIOS.tsx

```diff
@@ -109,55 +109,55 @@ const BrowserPermissionDeniedIOSContainer = (props: BrowserPermissionDeniedIOSPr
         <Stack styles={mergeStyleSets(iOSImageContainer, { root: { background: theme.palette.neutralLighter } })}>
           <Image
             src={imageSource ?? ''}
-            alt={strings.imageAltText}
+            alt={strings?.imageAltText}
             imageFit={ImageFit.centerContain}
             style={{ padding: '1.5rem' }}
           />
         </Stack>
       )}
       <Stack style={{ padding: '1rem', paddingBottom: '1.25rem' }}>
         <Stack styles={textContainerStyles}>
-          <Text styles={primaryTextStyles}>{strings.primaryText}</Text>
-          <Text styles={secondaryTextStyles}>{strings.secondaryText}</Text>
-          {strings.step1Text && (
+          <Text styles={primaryTextStyles}>{strings?.primaryText}</Text>
```

Comment:
Likewise here, now these strings are all optional they should follow the same pattern https://github.com/Azure/communication-ui-library/pull/2434/files#diff-86f3474befa4ad506ce8c7c7f9d1de9345eb84c5a14726dc9935af0180f0ccdcR91